src/strategies/python.ts

```diff
@@ -154,4 +157,27 @@ export class Python extends BaseStrategy {
   protected initialReleaseVersion(): Version {
     return Version.parse('0.1.0');
   }
+
+  protected async buildReleaseNotes(
+    conventionalCommits: ConventionalCommit[],
+    newVersion: Version,
+    newVersionTag: TagName,
+    latestRelease?: Release,
+    commits?: Commit[]
+  ): Promise<string> {
+    const releaseNotes = await super.buildReleaseNotes(
+      conventionalCommits,
+      newVersion,
+      newVersionTag,
+      latestRelease,
+      commits
+    );
+    return (
+      releaseNotes
+        // Remove links in version title line and standardize on h2
+        .replace(/^###? \[([\d.]+)\]\([^)]*\)/gm, '## $1')
```

Comment:
Note: This will remove the link in the version header (which I think normally points at a diff on github). We wanted that removed for Ruby, but make sure you also want it removed for Python, otherwise you should modify this logic accordingly.

Reply:
I've decided to only drop the link in the version header as there are still links to individual commits if folks were interested. I also think that the version header being a link is sort of a confusing UX experience (should you be clicking on those? Are these really the kind of information you want to look at versus just a summary of the changes?) and could confuse the users and lure them into clicking it. It was indeed intentional.

Reply:
The link is a quick link to the commits between the new release and the previous release and is helpful. Also, note that the `python` strategy is general and used by more than Google's python client libraries (Ruby's change is in `ruby-yoshi` strategy).

Reply:
Sounds good. Made updates to preserve the commit link.